src/App.js

```diff
@@ -31,24 +36,29 @@ import MyGroups from "./Path/MyGroups";
 import VerifyEmail from "./Path/VerifyEmail";
 import AccountSettings from "./Path/AccountSettings";
 import AllDevices from "./Path/AllDevices";
-
 import PasswordForgot from "./Path/PasswordForgot";
 import PasswordUpdate from "./Path/PasswordUpdate";
 import HeidiLogo from "./assets/HEIDI_Logo.png";
 import "./i18n";
-
 import ViewPost from "./Path/Forums/ViewPost";
 import GroupMembers from "./Path/MyGroups/GroupMembers";
 import MemberRequests from "./Path/MyGroups/MemberRequests";
 import ReportedPosts from "./Path/MyGroups/ReportedPosts";
-import IFrame from "./Path/Listings/IFrame.jsx";
-import IFrameListing from './Path/Listings/IFrameListing.jsx'
-import Modal from 'react-modal';
-Modal.setAppElement('#root');
+import IFrame from "./Path/Listings/IFrame";
+import IFrameListing from "./Path/Listings/IFrameListing.jsx";
+import MyAppointments from "./Path/AppointmentBooking/MyAppointments";
+import AppointmentsUserCreated from "./Path/AppointmentBooking/AppointmentsUserCreated";
+import BookMyAppointments from "./Path/AppointmentBooking/BookMyAppointments";
+import MyBookings from "./Path/AppointmentBooking/MyBookings";
+
+import Modal from "react-modal";
+Modal.setAppElement("#root");
 
 const App = () => {
   const isForumEnabled = process.env.REACT_APP_ENABLE_FORUM === "True";
   const inFrame = process.env.REACT_APP_INFRAME === "True";
+  const frontendVersion = process.env.REACT_APP_FORNTENDVERSION;
```

Comment:
Here add process.env.REACT_APP_FORNTENDVERSION ||  '1' so that by default it is set to 1

Reply:
Done